h/static/scripts/test/root-thread-test.js

```diff
@@ -199,30 +200,52 @@ describe('rootThread', function () {
       assert.isDefined(threadFilterFn({annotation: annotation}));
     });
 
-    it('generates a thread filter function to match notes', function () {
+    it('filter matches only notes when Notes tab is selected', function () {
       fakeBuildThread.reset();
-      fakeBuildThread.annotation = {target: [{}]};
 
       fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
-        {selectedTab: 'note'});
+        {selectedTab: uiConstants.TAB_NOTES});
 
       rootThread.thread(fakeAnnotationUI.state);
       var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
 
-      assert.isTrue(threadFilterFn(fakeBuildThread));
+      assert.isTrue(threadFilterFn({annotation: {target: [{}]}}));
     });
 
-    it('generates a thread filter function for annotations, when all annotations are of type notes', function () {
+    it('filter matches only orphans when Orphans tab is selected', function () {
       fakeBuildThread.reset();
-      fakeBuildThread.annotation = {target: [{}]};
 
       fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
-        {selectedTab: 'annotation'});
+        {selectedTab: uiConstants.TAB_ORPHANS});
 
       rootThread.thread(fakeAnnotationUI.state);
       var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
 
-      assert.isFalse(threadFilterFn(fakeBuildThread));
+      assert.isTrue(threadFilterFn({annotation: {$orphan: true}}));
+    });
+
+    it('filter matches only notes when Annotations tab is selected', function () {
+      fakeBuildThread.reset();
+
+      fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
+        {selectedTab: uiConstants.TAB_ANNOTATIONS});
+
+      rootThread.thread(fakeAnnotationUI.state);
+      var threadFilterFn = fakeBuildThread.args[0][1].threadFilterFn;
+
+      assert.isFalse(threadFilterFn({annotation: {target: [{}]}}));
+    });
+
+    it('filter matches only orphans when Annotations tab is selected', function () {
```

Comment:
The description here isn't the behavior we want either. We have a test above labeled "filter matches only orphans when Orphans tab is selected" which states the opposite user behavior.

From a user's POV there are two cases:

- When the orphans flag is disabled, the Annotations tab should display annotations and orphans
- When the orphans flag is enabled, the Annotations tab should display only annotations

I think it would be OK to combine the first two cases (orphans flag disabled) into one test with two assertions.